roles/edpm_libvirt/tasks/post-install.yml

```diff
@@ -57,6 +73,23 @@
         user: "{{ cluster_user_map[item] }}"
       loop: "{{ cluster_fsid_map.keys() | list }}"
 
+    # nova can pass alternate ceph.conf paths to libvirt for nova provisioned rbd volumes
+    # but not for cinder provisioned rbd volumes so we need to copy the ceph.conf files
+    # to /etc/ceph so qemu can find them
+    - name: Copy found_confs to /etc/ceph for qemu
+      become: true
+      ansible.builtin.copy:
+        src: "{{ item.path }}"
+        dest: "/etc/ceph/{{ (item.path | basename) }}"
+        owner: root
+        group: root
+        mode: "0644"
```

Comment:
please don't set this world read. @fultonj WDYT?

Reply:
again world reabale is what rhel would do by default
we can make it stricter but when we do we risk breaking things by deviating form the norm.

by default file in /etc are often 0644

but yes i can update his if @fultonj prefers a different value.